src/main/java/com/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java

```diff
@@ -54,6 +54,11 @@ public CommandType getType() {
         return CommandType.INFORMATIONAL;
     }
 
+    @Override
+    public Permission getPermission() {
+        return Permission.SERVER_INFO_COMMAND;
```

Comment:
It begins, but yegh why is permissions stuff in this pr

Reply:
Oh 🤔